src/async.c

```diff
@@ -63,59 +65,58 @@ static unsigned int callbackHash(const void *key) {
 }
 
 static void *callbackValDup(void *privdata, const void *src) {
-    ((void) privdata);
+    ((void)privdata);
     valkeyCallback *dup;
 
     dup = vk_malloc(sizeof(*dup));
     if (dup == NULL)
         return NULL;
 
-    memcpy(dup,src,sizeof(*dup));
+    memcpy(dup, src, sizeof(*dup));
     return dup;
 }
 
-static int callbackKeyCompare(void *privdata, const void *key1, const void *key2) {
+static int callbackKeyCompare(void *privdata, const void *key1,
+                              const void *key2) {
     int l1, l2;
-    ((void) privdata);
+    ((void)privdata);
 
     l1 = sdslen((const sds)key1);
     l2 = sdslen((const sds)key2);
-    if (l1 != l2) return 0;
-    return memcmp(key1,key2,l1) == 0;
+    if (l1 != l2)
+        return 0;
+    return memcmp(key1, key2, l1) == 0;
 }
 
 static void callbackKeyDestructor(void *privdata, void *key) {
-    ((void) privdata);
+    ((void)privdata);
     sdsfree((sds)key);
 }
 
 static void callbackValDestructor(void *privdata, void *val) {
-    ((void) privdata);
+    ((void)privdata);
     vk_free(val);
 }
 
 static dictType callbackDict = {
-    callbackHash,
-    NULL,
-    callbackValDup,
-    callbackKeyCompare,
-    callbackKeyDestructor,
-    callbackValDestructor
+    callbackHash,          NULL,
+    callbackValDup,        callbackKeyCompare,
+    callbackKeyDestructor, callbackValDestructor,
 };
```

Comment:
Yeah, this is a bit ugly, but iut's in very few places so I think it's OK.

We can maybe avoid it if make some manual changes to the code, like if we spell out the fields explicitly,

```C
    .hashFunction = callbackHash,
    .keyDup = NULL,
    ...
```

Reply:
If we use clang-format to check code format of the manual changes, it will report error. Or other folks run clang-format codes, it will be re-formatted. So I guess keeping the few places would be acceptable.